src/s3b_the_roach_market.tw2

```diff
@@ -344,3 +357,46 @@ I’d found this same box on this same table as a kid, and snatched it up immedi
 Honestly I was kinda hurt to see the box still on the table, but... whatever. Today’s ninjas had no fuckin’ taste.
 
 [[(back)|RM-junkheap-goodshit]]
+
+(set: $junkHeapItemsSeen to it +1)
+
+::RM-junkheap-flashback
+I walked up to the scooter and just took it in for a bit. It wasn't in good shape, nothing in this pile could say that.
```

Comment:
Comma splice: "It wasn't in good shape, nothing in this pile could say that."

Suggestions:
"It wasn't in good shape. Then again, nothing in this pile was."
"It wasn't in good shape, but then again, nothing in this pile was."

Reply:
Fixed